aspnetcore/client-side/angular/sample/AngularJSSample/src/AngularJSSample/Controllers/Api/PersonController.cs

```diff
@@ -16,7 +18,11 @@ public JsonResult GetPeople()
                 new Person { Id = 1, FirstName = "Bob", LastName = "Parker" }
             };
 
-            return Json(people);
+            // force the Json serializer to keep the case of the model properties
+            var serializerSettings = new JsonSerializerSettings();
+            serializerSettings.ContractResolver = new DefaultContractResolver();
+
```

Comment:
In principle I would support that idea, but I think that even if we change the variable names in the bindings to match the names elsewhere, we should still include that change (which would be effectively an identity operation in that case) with that comment to avoid confusion for new users who might not be aware that the code is converting the fields to camelCase without explicit warning.